Lib/unittest/test/test_discovery.py

```diff
@@ -832,40 +832,41 @@ def _import(packagename, *args, **kwargs):
             sys.modules[packagename] = package
             return package
 
-        def cleanup():
-            builtins.__import__ = orig_import
-        self.addCleanup(cleanup)
-        builtins.__import__ = _import
-
         _find_tests_args = []
         def _find_tests(start_dir, pattern, namespace=None):
             _find_tests_args.append((start_dir, pattern))
             return ['%s/tests' % start_dir]
 
         loader._find_tests = _find_tests
         loader.suiteClass = list
-        suite = loader.discover('package')
+
+        with unittest.mock.patch('builtins.__import__', _import):
+            # since loader.discover() can modify sys.path, restore it when done
```

Comment:
Capitalize the start of the sentence and end it in a period (same for other comments).